### PR TITLE
[PT-1271] Nullable in required

### PR DESCRIPTION
**Pull request by newmana**
Similar effect but put nullable in required.

image

### CODE REVIEW

File: src/Form/Field.elm [hunk 1]
```diff
@@ -288,17 +289,25 @@ type alias CommonFieldProperties =
...
+    StringFieldProperties
+        { tipe : FieldType.SimpleFieldType
+        }
 
 
 {-| -}
 type alias SelectFieldProperties =
-    StringFieldProperties { default : Maybe String, options : List Option.Option }
+    StringFieldProperties
+        { default : Maybe String
```
Comment (ljnicol) [inline src/Form/Field.elm:308]: Looks like some formatting changes - is this an elm-format thing?

File: src/Form/View/Input.elm [hunk 2]
```diff
@@ -56,7 +57,7 @@ label field disabled =
     HtmlExtra.viewIf (not (Field.isCheckbox field)) <|
         Html.label [ HtmlAttributes.class "label" ]
             [ Html.text (Field.getLabel field)
-            , HtmlExtra.viewIf (not (Field.isRequired field) && not disabled) <|
+            , HtmlExtra.viewIf (not (Field.isRequired field == Required.Yes) && not disabled) <|
```
Comment (ljnicol) [inline src/Form/View/Input.elm:60]: I wonder if we could add an element to show its nullable?

**Review by ljnicol — APPROVED** [review]
Looks good - only question is what happens when a field like Radio has "nullable". Does that mean that a "" as an option will be set to null?

**Comment by newmana** [comment]
The radio button's behaviour is as before - the only field with a nullable mapping is the string one.

**Pull request merged**